apps/publish/content/common.py

```diff
@@ -538,28 +542,46 @@ def apply_kill_override(self, item, updates):
         except Exception:
             logger.exception('Failed to apply kill header template to item {}.'.format(item))
 
-    def _refresh_associated_items(self, original):
+    def _refresh_associated_items(self, original, updates={}, publish=False):
```

Comment:
imo would be better to have publish_associated_items, it would be more consistent with packages,
now it's a bit misleading when refresh does publishing

Reply:
You are right it is misleading refresh does publishing. Those 2 should be separate.